Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/Commands.cpp

```diff
@@ -588,6 +588,8 @@ namespace EMStudio
             {
                 return false;
             }
+
+            animGraph->SetDirtyFlag(false);
```

Comment:
this fix is in conflict with this one https://github.com/o3de/o3de/pull/5936
5936 also check the update flag which is the proper way. I suggest take this change out and wait for 5936 submitted.